lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -75,10 +75,8 @@ func (h *StorageHandlersImpl) Configure(api *operations.PortLayerAPI, handlerCtx
 	// expensive metadata lookups.
 	h.imageCache = spl.NewLookupCache(ds)
 
-	// add the volume stores
-	if err = h.configureVolumeStores(op, handlerCtx); err != nil {
-		log.Panicf("Cannot instantiate volume stores: %s", err.Error())
-	}
+	// add the volume stores, errors are logged within this function.
+	h.configureVolumeStores(op, handlerCtx)
```

Comment:
No this was originally the problem, we want to make sure that all volume stores were established at vic-machine create time. in order to do this we must allow the portlayer to establish all possible volume-stores. Vic-machine will then compare the list of returned volume-stores during a `docker info` call and compare that to the desired volume store calls. The portlayer should not be aborted just because an nfs store was not reachable. The datastores are checked at vic-machine time, but as previously discussed we cannot do that for nfs stores.